src/components/BottomBar/TimePicker.jsx

```diff
@@ -112,8 +120,15 @@ function TimePicker() {
   }
 
   function timeLabel() {
-    return time && time.toUTCString();
-  }
+    if (time) {
+      try {
+        return time.toUTCString();
+      } catch {
+        return time;
+      }
+    }
+    return time;
```

Comment:
This will result in "Undefined" being used as the time label, if it ever happens.  But maybe that's not a problem

Reply:
going to leave it since that was the previous behavior, and I'm not sure what we would want the improvement to be